db/structure.sql

```diff
@@ -0,0 +1,276 @@
+SET statement_timeout = 0;
```

Comment:
We also shouldn't move away from a schema.rb unless we absolutely have to.

Reply:
Reverted this change as well